test/Elastic.Apm.Tests/SystemInfoHelperTests.cs

```diff
@@ -44,25 +45,21 @@ public void ParseKubernetesInfo_ShouldReturnNull_WhenNoEnvironmentVariablesAreSe
 			system.Kubernetes.Should().BeNull();
 		}
 
+		public struct CGroupTestData
+		{
+			public string GroupLine;
+			public string ContainerId;
+			public string PodId;
+		}
+
 		[Theory]
-		// underscores
-		[InlineData("1:name=systemd:/kubepods.slice/kubepods-burstable.slice/" +
-			"kubepods-burstable-pod90d81341_92de_11e7_8cf2_507b9d4141fa.slice/" +
-			"crio-2227daf62df6694645fee5df53c1f91271546a9560e8600a525690ae252b7f63.scope",
-			"2227daf62df6694645fee5df53c1f91271546a9560e8600a525690ae252b7f63", "90d81341-92de-11e7-8cf2-507b9d4141fa")]
-		// openshift form
-		[InlineData("9:freezer:/kubepods.slice/kubepods-pod22949dce_fd8b_11ea_8ede_98f2b32c645c.slice" +
-			"/docker-b15a5bdedd2e7645c3be271364324321b908314e4c77857bbfd32a041148c07f.scope",
-			"b15a5bdedd2e7645c3be271364324321b908314e4c77857bbfd32a041148c07f",
-			"22949dce-fd8b-11ea-8ede-98f2b32c645c")]
-		// ubuntu cgroup
-		[InlineData("1:name=systemd:/user.slice/user-1000.slice/user@1000.service/apps.slice/apps-org.gnome.Terminal" +
-			".slice/vte-spawn-75bc72bd-6642-4cf5-b62c-0674e11bfc84.scope", null, null)]
-		// AWS ECS cgroup
-		[InlineData("1:name=systemd:/ecs/03752a671e744971a862edcee6195646/03752a671e744971a862edcee6195646-4015103728",
-			"03752a671e744971a862edcee6195646-4015103728", null)]
-		public void ParseKubernetesInfo_FromCGroupLine(string line, string containerId, string podId)
+		[JsonFileData("./TestResources/json-specs/cgroup_parsing.json", typeof(CGroupTestData))]
+		public void ParseKubernetesInfo_FromCGroupLine(string name, CGroupTestData data)
```

Comment:
`string name` is unused in the method and Rider shows a warning. On the other hand we can't remove it, because it's in the data that gets passed into the method, plus all the debug output shows these parameters and that's super useful, because if it fails, we know which test-case fails.

Adding `#pragma warning disable xUnit1026` (disabling the warning) would maybe just make the code more noisy, I'm not sure if it's better than just accepting as is.

This is probably fine as is, I'm just adding here what I was thinking.

Reply:
Thanks, I actually missed that warning.
I guess it's really time we get our warnings down to zero in https://github.com/elastic/apm-agent-dotnet/issues/1823 and flag them as errors from then on.
I will add the suggested `#pragma`.